README.md

```diff
@@ -125,16 +141,37 @@ module load java matlab conda2
 # Get the latest version of the pipeline
 nextflow pull labsyspharm/mcmicro-nf
 
-# All previous commands require an additional `-profile O2` flag
-nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-001 -profile O2
-nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-002 --tma -profile O2
+# All previous commands require an additional `-profile O2` flag and must be run from a compute node
+srun -p priority -t 0-12 --mem 8G nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-001 -profile O2
+srun -p priority -t 0-12 --mem 8G nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-002 --tma -profile O2
+```
+
+In the above, `-t 0-12 --mem 8G` requests 12 hours of compute time and 8GB of memory from the O2 cluster. To avoid running over on your disk quota, it is also recommended to use `/n/scratch2` for holding the `work/` directory:
+
+```
+srun -p priority -t 0-12 --mem 8G \
+  nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-001 -profile O2 -w /n/scratch2/$USER/work
 ```
 
-To avoid running over on your disk quota, it is recommended to use `/n/scratch2` for holding the `work/` directory:
+An alternative to the above `srun` command is to compose an `sbatch` script that encapsulates resource requests, module loading and the `nextflow` command into a single entity. Create a `submit_mcmicro.sh` file based on the following template:
 
 ```
-nextflow run labsyspharm/mcmicro-nf --in path/to/exemplar-001 -profile O2 -w /n/scratch2/$USER/work
+#!/bin/sh
+#SBATCH -p medium
+#SBATCH -J nextflow_O2              
+#SBATCH -o run.o
+#SBATCH -e run.e
+#SBATCH -t 0-12:00
+#SBATCH --mem=8G
+#SBATCH --mail-type=END         # Type of email notification- BEGIN,END,FAIL,ALL
+#SBATCH --mail-user=user@university.edu   # Email to which notifications will be sent
+
+module load java matlab conda2
+/home/$USER/bin/nextflow labsyspharm/mcmicro-nf --in /n/scratch2/$USER/exemplar-001 -profile O2 -w /n/scratch2/$USER/work
 ```
+replacing relevant fields (e.g., `user@university.edu`) with your own values.
+
+The pipeline run can then be kicked off with `source submit_mcmicro.sh`.
```

Comment:
This needs to be `sbatch submit_mcmicro.sh`! Otherwise it will just run nextflow in the current shell and not submit an actual job.

Reply:
Whoops!